### PR TITLE
Modified the servlet file in the portfolio directory as directed in the servlet tutorial

### PR DESCRIPTION
**Pull request by gracieorozco**
I modified the DataServlet.java file in the portfolio directory as directed in the servlet tutorial. This pull request is one out of five pull requests required for week 2.

### CODE REVIEW

**Comment by gracieorozco** [comment]
I modified the index.html, script.js, and DataServlet.java files to use fetch() and text() to receive data through promise objects. This pull request is the second out of five pull requests required for week 2. 

Also, I had a few questions regarding week 2 and the project in general. If this is not the place to ask these questions, please let me know where I should ask for next time.

1. For the servlet files, is it important to understand how init() and doGet() functions exactly work? I understand the main concept of the servlet being a server, but the servlet tutorial did not explain every component of the functions. Do you know if there are any document/resources I can use to better understand how and why it is structured the way it is?

2. When using .then for a promise, does the program automatically input the result into the variable for the next upcoming function? If so, how does it know? 
For example: 
const response = fetch('/data'); 
response .then(handleResponse);  
// Does the .then automatically place the response const into the next function?
// Also, would this work for multiple variables, or just one?
function handleResponse(response) ...

3. What is the preferred approach to using promise objects? I understand all the different type of implementations, but I would like what is preferred regarding promise objects, promises formatted with arrow functions, or await formatting.

4. Would it be okay to create new pages within my portfolio? I would like to organize the content I create for weeks into different pages.

**Comment by KaylaNguyen** [comment]
Hi Gracie!
I see you have 2 commits under 1 PR. For the purpose of this week assignment, can you make them 2 separate PRs instead?

Regarding questions, hangout chat for our team or technical/project chat would work best! Let's move the discussion there.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -27,6 +27,8 @@
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    // below command was used in the servlet tutorial
+    // response.getWriter().println("<h1>Hello Gracie!</h1>");
```
Comment (KaylaNguyen) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: Remove unused code.

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -28,3 +28,17 @@ function addRandomFact() {
   const factContainer = document.getElementById('fact-container');
   factContainer.innerText = fact;
 }
+
+/**
+ * Uses fetch() to obtain data from DataServlet.java
+ */
+ function obtain_fetch_data() {
+     fetch("/data")
```
Comment (KaylaNguyen) [inline portfolio/src/main/webapp/script.js]: Very nice!

**Pull request merged**